### ISSUE TITLE
pa11y Error

### ISSUE THREAD

**Issue by nilswindisch** [nilswindisch/nilswindisch.github.io#2]
```
WCAG2AA.Principle1.Guideline1_1.1_1_1.H30.2 1
First result: Img element is the only content of the link, but is missing alt text. The alt text should describe the purpose of the link.

Selector: html > body > div:nth-child(2) > div:nth-child(1) > header > div:nth-child(1) > a 
Context: <a href="/" title="Nils Windisch"><img src="img/nilswindisch.svg"...</a>
```
